pkg/helpers/eval_query.go

```diff
@@ -0,0 +1,43 @@
+package helpers
+
+import (
+  "reflect"
+  "time"
+
+  "github.com/altinity/clickhouse-grafana/pkg/eval"
+)
+
+// NewEvalQuery creates a fully populated EvalQuery using direct field access on request
+func NewEvalQuery(request interface{}, from, to time.Time) eval.EvalQuery {
```

Comment:
move this code back to pkg/eval

NewEvalQuery this is not a helper this is contructor
this shall be in eval packager